Makefile

```diff
@@ -139,10 +139,9 @@ systest: $(CLIENT_CERT)
 
 
 # Removes the virtual environment created via this makefile,
-# NOTE: since venv is a temporary directory, this will only remove files if
-# 		run after a target, e.g. "make systest clean"
+# NOTE: this will remove all previous virtual environments
 .PHONEY: clean
 clean:
-	@echo "Removing local test virtual environment"; $(BLR)
-	rm -rf $(VENV)
+	@echo "Removing local test virtual environments"; $(BLR)
+	rm -rf /tmp/caramel-test.*
```

Comment:
Should have a  " -- " between as well, and even then the "*" is scary, as anyone else can create various files that have that name. Better to quote the variable VENV properly and remove only what we know we created.


try creating a file named `/tmp/caramel-test.a ..`  or similar.   ( Do not recommend)

Reply:
I understand your thinking and its probably correct, but my thought was that if you miss "clean" on your first few runs, then there is no real "easy/lazy" way to remove them.